tests/acceptance/config/behat.yml

```diff
@@ -13,7 +13,7 @@ default:
           ldapGroupsOU: TestGroups
           ldapInitialUserFilePath: /../../config/ldap-users.ldif
       contexts:
-        - RevaContext:
+        - SpacesContext:
```

Comment:
We need to mention a local context first in this list, otherwise Behat complains. So mention SpacesContext.
This suite does not happen to use any steps in a local context - but we need to mention one.